.azure/Makefile

```
@@ -17,3 +17,5 @@ test:
 
 clean:
 	rm -f azure-pipelines.yml
+
+test:
```

checkmake (0.3.2)

[warning] Target "test" defined multiple times (lines 16 and 21).

(uniquetargets)